modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/TxManagerImpl.java

```diff
@@ -198,6 +203,8 @@ public TxManagerImpl(
                 new NamedThreadFactory("tx-async-cleanup", LOG));
 
         orphanDetector = new OrphanDetector(clusterService.topologyService(), replicaService, placementDriver, lockManager, clock);
+
+        txUnlockManager = new TxUnlockManager(clusterService, this, lockManager, placementDriver, clock);
```

Comment:
Why we need the TxUnlockManager? Especially because we already have LockManager with all that acquire and release operations. From my point of view adding TxUnlockManager will bring event more mess.